### PR TITLE
Update UrlFetch sample

### PR DESCRIPTION
**Pull request by averikitsch**
Fixes Issue #1225

* Fixed validation for id
* Updated HTTP content type
* Add url stream handler to pom

### CODE REVIEW

**Comment by lesv** [comment]
I notice that Kokoro isn't happy

File: appengine-java8/urlfetch/src/main/webapp/WEB-INF/appengine-web.xml [hunk 1]
```diff
@@ -2,4 +2,5 @@
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
   <runtime>java8</runtime>
   <threadsafe>true</threadsafe>
+  <url-stream-handler>urlfetch</url-stream-handler>
```
Comment (lesv) [inline appengine-java8/urlfetch/src/main/webapp/WEB-INF/appengine-web.xml]: This is only needed if you wish to use our service to do it, why do you need this?

Comment (averikitsch) [reply appengine-java8/urlfetch/src/main/webapp/WEB-INF/appengine-web.xml]: I read our documentation/error wrong...

**Comment by lesv** [comment]
LGTM

**Pull request merged**